tests/test_write_run_directory.py

```diff
@@ -129,3 +129,12 @@ def test_cached_diag_table_is_not_redownloaded(self):
         with tempfile.TemporaryDirectory() as rundir:
             fv3config.write_run_directory(config, rundir)
         assert os.path.getmtime(cache_filename) == modification_time
+
+    def test_rundir_contains_fv3config_yml(self):
+        config = copy.deepcopy(DEFAULT_CONFIG)
+        config[
+            "diag_table"
+        ] = "gs://vcm-fv3config/config/diag_table/default/v1.0/diag_table"
+        with tempfile.TemporaryDirectory() as rundir:
```

Comment:
May as well remove the "special" diag table to make the configuration more "standard" (assuming you copied this from a remote diag table test).
```suggestion
        with tempfile.TemporaryDirectory() as rundir:
```